test/nn/test_load_state_dict.py

```diff
@@ -362,7 +365,162 @@ def test_load_state_dict_warn_assign(self):
             m.load_state_dict(state_dict)
 
 
+class MyLoadTensor(torch.Tensor):
+
+    # enable deepcopy
```

Comment:
deepcopy doesn't work without that?

Reply:
I think  `new_empty` needs to be implemented as `__deepcopy__` goes through this path for non-wrapper

https://github.com/pytorch/pytorch/blob/9fe3693bbb9494f57febebe0d6aefa9da081b16a/torch/_tensor.py#L172-L188

The override seemed to error if within the `__torch_function__` but that might just be a bug on my end, will retry.